### ISSUE TITLE
issue/enh: incorrect input markdown

### ISSUE THREAD

**Issue by HeyThisWasRandomlyMade** [open-webui/open-webui#6580]
# Bug Report

Installation Method

Docker

## Environment

- **Open WebUI Version:** v0.3.35

- **Operating System:** kubuntu 24.04

**Confirmation:**

- [ x] I have read and followed all the instructions provided in the README.md.
- [ x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

It should either format it into italics correctly or just not format any text in the message box into markdown at all until after you send the message.

## Actual Behavior:

The text is bolded instead. It won't paste text with markdown in it on mobile. On mobile you cannot stop bold text from appearing after bolding it.

## Description

**Bug Summary:**
Issue: Markdown *italics* is formated into **bold** in the message box. It interprets the asterisk into a double asterisk, making it bold. This also makes it impossible to paste something into the text box on mobile with markdown formatting already in it. If you're on mobile, it's impossible to stop text being bolded, meaning you have to edit the message after or delete all your work.

## Reproduction Details

**Steps to Reproduce:**

1. Go to the webui.
2. Try to type anything down in italics.
3. It's formated into bold text instead, making it impossible to get it without editing the message afterwards.
4. If you're on mobile, anytime you paste anything with markdown in it, it won't work unless you edit the message and paste.
5. If you are on mobile, you cannot stop the bolding. This leaves you with a never ending string of bolded text.

Basically I'd like to fix this issue and make it an option to disable markdown in the text box and format it into it after sending the message or revert the change if it was unintended.

**Comment by ProjectMoon** [comment]
Similar happens with predefined prompts. They're put into the textbox in rich text mode and then rendered incorrectly on output. Notable case is --- horizontal rule, which winds up as a H1 header instead because the blank lines are removed.

**Comment by tjbck** [comment]
<img width="923" alt="image" src="https://github.com/user-attachments/assets/9cc222ef-1b28-48a1-aba6-c24761ec8e9b">

You can most definitely toggle it off the rich text input from the settings. I believe some of the issues here are resolved with the latest dev, testing wanted here!

Closing in favour of #6973
